Frameworks/QuartzCore/CALayer.mm

```diff
@@ -1583,7 +1581,7 @@ - (void)setBackgroundColor:(CGColorRef)color {
     // is updated almost immediately.By setting the priv->background first,
     // we can ensure both states are kept in sync which is useful when performing
     // automated testing.
-    [CATransaction _setPropertyForLayer:self name:@"backgroundColor" value:(NSObject*)color];
+    [CATransaction _setPropertyForLayer:self name:@"backgroundColor" value:(NSObject*)[UIColor colorWithCGColor:color]];
```

Comment:
consider them dead

---
In reply to: [113286201](https://github.com/Microsoft/WinObjC/pull/2538#discussion_r113286201) [](ancestors = 113286201)